### PR TITLE
monitor: Fix mismatching frontend service debug trace types

### PR DESCRIPTION
**Pull request by christarazi**
It seems they were missed in commit 308d67fb19a ("lb: Use more neutral
terminology"). However, this was harmless as these debug types are no
longer used as of commit f3c40a729d ("datapath: Do not log if svc is not
found"). This commit is essentially cosmetic.

Fixes: 308d67fb19a ("lb: Use more neutral terminology")

Signed-off-by: Chris Tarazi <chris@isovalent.com>

### CODE REVIEW

**Comment by christarazi** [comment]
test-me-please

File: pkg/monitor/datapath_debug.go [hunk 1]
```diff
@@ -305,9 +305,9 @@ func (n *DebugMsg) Message() string {
 		return fmt.Sprintf("Packet hash=%d (%#x), selected_service=%d", n.Arg1, n.Arg1, n.Arg2)
 	case DbgRRBackendSlotSel:
 		return fmt.Sprintf("RR backend slot selection hash=%d (%#x), selected_service=%d", n.Arg1, n.Arg1, n.Arg2)
-	case DbgLb6LookupMaster:
+	case DbgLb6LookupFrontend:
 		return fmt.Sprintf("Master service lookup, addr.p4=%x key.dport=%d", n.Arg1, byteorder.NetworkToHost16(uint16(n.Arg2)))
```
Comment (brb) [inline pkg/monitor/datapath_debug.go]: Shall we change the debug msg too?

**Review by brb — COMMENTED** [review]
Good catch! Left a comment about the debug msg content.

File: pkg/monitor/datapath_debug.go (hunk 1 above)
Comment (christarazi) [reply pkg/monitor/datapath_debug.go]: Yes good catch

**Review by brb — APPROVED** [review]
Thanks!

**Comment by christarazi** [comment]
test-me-please

**Comment by christarazi** [comment]
test-me-please

**Review by kkourt — APPROVED** [review]
Changes look good to me, thanks!

I'll mark `ready-to-merge` since  the patch does not affect any run-time behavior other than monitor messages.

**Pull request merged**